ta/avb/user_ta.mk

```diff
@@ -1 +1,2 @@
 user-ta-uuid := 023f8f1a-292a-432b-8fc4-de8471358067
+user-ta-version := $(if $(CFG_TA_VERSION),$(CFG_TA_VERSION),0)
```

Comment:
`user-ta-version := 0` should be enough here

Reply:
ok